utils/utils.go

```diff
@@ -487,3 +487,27 @@ func isUrlAccessible(url string) bool {
 	}
 	return resp != nil && resp.StatusCode == http.StatusOK
 }
+
+// GetAllJsonsInString searches for a pattern of json objects in the input parameter
+// Returns a slice of these objects
```

Comment:
Let's an example to this description, to help the reader quickly understand the functionality.